terraform/modules/r53/main.tf

```diff
@@ -0,0 +1,283 @@
+}
+
+resource "aws_route53_record" "im-ipv6" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "im.${aws_route53_zone.hashbang.name}"
+    type = "AAAA"
+    ttl  = "1800"
+    records = ["2604:a880:1:20::db8:5001"]
+}
+resource "aws_route53_record" "im-ipv4" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "im.${aws_route53_zone.hashbang.name}"
+    type = "A"
+    ttl  = "1800"
+    records = ["107.170.254.61"]
+}
+
+resource "aws_route53_record" "www-ipv6" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "www.${aws_route53_zone.hashbang.name}"
+    type = "AAAA"
+    ttl  = "1800"
+    records = ["2604:a880:800:10::1909:a001"]
+}
+resource "aws_route53_record" "www-ipv4" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "www.${aws_route53_zone.hashbang.name}"
+    type = "A"
+    ttl  = "1800"
+    records = ["104.131.13.197"]
+}
+
+resource "aws_route53_record" "hashbang-ipv6" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "${aws_route53_zone.hashbang.name}"
+    type = "AAAA"
+    ttl  = "1800"
+    records = ["2604:a880:800:10::1909:a001"]
+}
+resource "aws_route53_record" "hashbang-ipv4" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "${aws_route53_zone.hashbang.name}"
+    type = "A"
+    ttl  = "1800"
+    records = ["104.131.13.197"]
+}
+
+resource "aws_route53_record" "voip-ipv6" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "voip.${aws_route53_zone.hashbang.name}"
+    type = "AAAA"
+    ttl  = "1800"
+    records = ["2604:a880:2:d0::17d:6001"]
+}
+resource "aws_route53_record" "voip-ipv4" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "voip.${aws_route53_zone.hashbang.name}"
+    type = "A"
+    ttl  = "1800"
+    records = ["138.68.57.139"]
+}
+
+resource "aws_route53_record" "nyc3-apps-ipv6" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "nyc3.apps.${aws_route53_zone.hashbang.name}"
+    type = "AAAA"
+    ttl  = "1800"
+    records = ["2604:a880:800:10::17cb:1001"]
+}
+resource "aws_route53_record" "nyc3-apps-ipv4" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "nyc3.apps.${aws_route53_zone.hashbang.name}"
+    type = "A"
+    ttl  = "1800"
+    records = ["104.236.85.7"]
+}
+
+// TXT
+resource "aws_route53_record" "hashbang-txt" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "_dmarc.${aws_route53_zone.hashbang.name}"
+    type = "TXT"
+    ttl  = "1800"
+    records = ["v=DMARC1; p=none; pct=100; ruf=kellerfuchs+dmarc@${aws_route53_zone.hashbang.name}; rua=mailto:re+niiphewtj3k@dmarc.postmarkapp.com; sp=none; aspf=s;"]
+}
+
+resource "aws_route53_record" "dmarc-txt" {
+    zone_id = "${aws_route53_zone.hashbang.zone_id}"
+    name = "${aws_route53_zone.hashbang.name}"
+    type = "TXT"
+    ttl  = "1800"
+    records = ["v=spf1 +mx -all"]
+}
+
+
+// ALIAS
+resource "aws_route53_record" "va1-alias" {
```

Comment:
@KellerFuchs no, it can't. Not sure what the best way to find out this traffic could be.